tests/spread/plugins/craft-parts/colcon/colcon-packages-ignore/task.yaml

```diff
@@ -18,6 +18,9 @@ prepare: |
   . "$TOOLS_DIR/snapcraft-yaml.sh"
   set_base "../snaps/$SNAP/snap/snapcraft.yaml"
 
+  . "$TOOLS_DIR/package-utils.sh"
```

Comment:
mind adding this to the foreach entries in the main spread.yaml?

Reply:
I can give it a shot, but I am scared that some tests might just break. I will let you know.

Reply:
@sergiusens, do you recommand me to break the test and to fix them later, or should I use the "apt packages breakpoint" only for the ROS tests like it is right now?